FirebaseAppDistribution/Sources/FIRAppDistributionAppDelegateInterceptor.m

```diff
@@ -71,16 +78,17 @@ - (void)appDistributionRegistrationFlow:(NSURL *)URL
   } else if (@available(iOS 11.0, *)) {
     _safariAuthenticationVC = [[SFAuthenticationSession alloc]
               initWithURL:URL
-        callbackURLScheme:@"com.firebase.appdistribution"
+        callbackURLScheme:callbackURL
         completionHandler:^(NSURL *_Nullable callbackURL, NSError *_Nullable error) {
           [self resetUIState];
-          NSLog(@"Testing: Sign in Complete!");
+          FIRFADInfoLog(@"Sign in complete using SFAuthenticationSession");
           // TODO (b/161538029): Map these errors to AppDistribution error codes
+
           completion(error);
         }];
 
     [_safariAuthenticationVC start];
-  } else {
+  } else if (@available(iOS 9.0, *)) {
```

Comment:
Should we raise an error in the else case? It should never happen but it may be useful to tell folks that they are out of luck.

Reply:
Good question. I thought about this too and left it as it to discuss. The podspec will not allow someone to try to build an app for < 9.0, so there is a technical constraint in place to prevent this from happening. We could just log or crash the app in this case, but I wasn't sure if it was worth it.

Reply:
👍  sounds good!